voluncheer/templates/profiles/profile.html

```diff
@@ -137,6 +137,28 @@ <h2>SAVED OPPORTUNITIES</h2>
                     {% endif %}
                 {% endfor %}
         </div>
+
+        <h2>PAST OPPORTUNITIES</h2>
+        <div class="card" id="saved_opportunities">
+            {% csrf_token %}
```

Comment:
It looks like we only need to use this in POST forms, there isn't any thing here that we would need to protect, unless I'm missing something. 

https://docs.djangoproject.com/en/4.2/howto/csrf/

Reply:
Deleted